trade-module/src/main/java/io/edpn/backend/trade/adapter/persistence/entity/mapper/MybatisPageInfoMapper.java

```diff
@@ -0,0 +1,17 @@
+package io.edpn.backend.trade.adapter.persistence.entity.mapper;
+
+import io.edpn.backend.trade.application.domain.PageInfo;
+import io.edpn.backend.trade.application.dto.persistence.entity.PersistencePageInfo;
+import io.edpn.backend.trade.application.dto.persistence.entity.mapper.PersistencePageInfoMapper;
+
+public class MybatisPageInfoMapper implements PersistencePageInfoMapper {
```

Comment:
this would mean breaking changes for all modules, one change in will ripple to all. the modules are not versioned independently in the modulith

Reply:
> this would mean breaking changes for all modules, one change in will ripple to all. the modules are not versioned independently in the modulith

Well, is there something keeping us from versioning them independently?